OpenSim/Simulation/Model/Blankevoort1991Ligament.cpp

```diff
@@ -158,51 +158,61 @@ double Blankevoort1991Ligament::getLengtheningSpeed(
 }
 
 double Blankevoort1991Ligament::getStrain(const SimTK::State& state) const {
```

Comment:
This is an example of how `CacheVariable<T>` can be used locally to reduce lookups. In this case, the map lookup + downcast only happens once (previous case, 3 lookups and 2 downcasts, potentially)